src/event-helper.js

```diff
@@ -101,7 +101,7 @@ export function isLoaded(element) {
  * Returns a promise that will resolve or fail based on the element's 'load'
  * and 'error' events. Optionally this method takes a timeout, which will reject
  * the promise if the resource has not loaded by then.
- * @param {!Element} element
+ * @param {!EventTarget} element
  * @param {number=} opt_timeout
  * @return {!Promise<!Element>}
```

Comment:
yeah, feel free to just get this green and merge. I cherry picked this PR into my types8 branch (just local to me) to continue typing changes w/o too much conflicts